client/components/gsuite/gsuite-price/index.jsx

```diff
@@ -24,8 +24,8 @@ export const hasDiscount = ( product ) => {
 	}
 
 	const currentTime = Date.now();
-	const startDate = new Date( product.sale_coupon.start_date );
-	const endDate = new Date( product.sale_coupon.expires );
+	const startDate = new Date( product.sale_coupon.start_date.replace( ' ', 'T' ) + 'Z' );
+	const endDate = new Date( product.sale_coupon.expires.replace( ' ', 'T' ) + 'Z' );
```

Comment:
I wonder if we shouldn't add a comment to explain what happens here as it's really not obvious, e.g:

```suggestion
	// Makes sure start and end dates are interpreted/parsed as UTC (e.g. '1995-12-17 13:24:00' to '1995-12-17 13:24:00Z')
	const startDate = new Date( product.sale_coupon.start_date.replace( ' ', 'T' ) + 'Z' );
	const endDate = new Date( product.sale_coupon.expires.replace( ' ', 'T' ) + 'Z' );
```

Reply:
Good suggestion! I've made the comment a bit longer, but have added more detail.